plugins/filter/gcp_kms_filters.py

```diff
@@ -36,12 +34,12 @@ def run(self, method, **kwargs):
             'projects': kwargs.get('projects', None),
             'scopes': kwargs.get('scopes', None),
             'locations': kwargs.get('locations', 'global'),
-            'auth_kind': kwargs.get('auth_kind', None),
+            'auth_kind': kwargs.get('auth_kind', 'application'),
             'service_account_file': kwargs.get('service_account_file', None),
             'service_account_email': kwargs.get('service_account_email', None),
         }
         if not params['scopes']:
-            params['scopes'] = ['https://www.googleapis.com/auth/cloudkms']
+            params['scopes'] = ['https://www.googleapis.com/auth/cloud-platform']
```

Comment:
this is technically a backwards-incompatible change (the default value of a field changing).

Is there a specific reason this needs to be done in this change?